src/main/java/com/exactpro/th2/common/schema/factory/CommonFactory.java

```diff
@@ -202,6 +221,162 @@ public static CommonFactory createFromArguments(String... args) {
         }
     }
 
+    public static CommonFactory createFromKubernetes(String namespace, String boxName) {
+
+        Resource<ConfigMap, DoneableConfigMap> boxConfigMapResource;
+        Resource<ConfigMap, DoneableConfigMap> rabbitMqConfigMapResource;
+        Resource<ConfigMap, DoneableConfigMap> cradleConfigMapResource;
+
+        ConfigMap boxConfigMap;
+        ConfigMap rabbitMqConfigMap;
+        ConfigMap cradleConfigMap;
+
+        Path grpcPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + ROUTER_GRPC_FILE_NAME);
+        Path rabbitMqPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + RABBIT_MQ_FILE_NAME);
+        Path cradlePath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + CRADLE_FILE_NAME);
+        Path mqPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + ROUTER_MQ_FILE_NAME);
+        Path customPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + CUSTOM_FILE_NAME);
+        Path prometheusPath = Path.of(System.getProperty("user.dir") + "/generated_configs/" + PROMETHEUS_FILE_NAME);
+        Path dictionaryPath = Path.of(System.getProperty("user.dir") + "/generated_configs/dictionary.json");
```

Comment:
Extract paths to constants. 
Use intermediate variable / const for method results (I mean the multiple `System.getProperty("user.dir")` call)
Avoid specifying file separator explicitly, please use the Path.of(**vararg**) method to build path according to file separator from operation system.